packages/bam-ppx/bam-ppx.0.3/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "A PPX deriving generators for OCaml types"
+description:
+  "Provides a way to automatically get generators for a given type"
+maintainer: ["François Thiré"]
```

Comment:
Would you mind adding a contact email here, and in the other packages? E.g.,

```
maintainer: ["François Thiré" <your.email@mail.server>]
```

Reply:
Done.

Reply:
Thank you!